cli/tests/tui/test_setup_panel.py

```
@@ -185,6 +186,105 @@ def test_local_observability_wizard_has_no_retired_audit_sink_flag() -> None:
+    profile_fields = _redaction_wizard_fields_for(
+        {
+            "@Action": "profile-set",
+            "@Custom Profile Name": "soc",
+            "--extends": "strict",
+            "--detector": "pii,secrets",
+            "--field-path": "hash",
+        }
+    )
+    profile_argv = build_wizard_args(SetupWizard.REDACTION, profile_fields)
+    assert profile_argv[:6] == ("setup", "redaction", "profile", "set", "soc", "--extends")
+    assert ("--field", "path=hash") == tuple(profile_argv[-4:-2])
```

Ruff (0.16.1)

[warning] Yoda condition detected

(SIM300)